desktop/core/src/desktop/lib/thrift_util.py

```diff
@@ -277,20 +281,25 @@ def return_client(self, conf, client):
       conf.update_coordinator_host(client.get_coordinator_host())
 
     self.pooldict[_get_pool_key(conf)].put(client)
+  def clear_client(self, conf):
```

Comment:
>   def clear_client(self, conf):
    self.pooldict.pop(_get_pool_key(conf), None)
  def check_pool_empty(self, conf):
    return self.pooldict.get(_get_pool_key(conf)) is None


Is this used?